packages/express/src/schematics/application/application.ts

```diff
@@ -27,7 +29,7 @@ function addTypes(options: NormalizedSchema): Rule {
 function addMainFile(options: NormalizedSchema): Rule {
   return (host: Tree) => {
     host.overwrite(
-      join(options.appProjectRoot, 'src/main.ts'),
+      join(options.appProjectRoot, options.js ? 'src/main.js' : 'src/main.ts'),
```

Comment:
I think the `toJS` rule takes care of this right?

Reply:
Should yeah, not sure why I changed this